src/components/pages/Analytics/TechStackGraph.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import axios from 'axios';
+import embed from 'vega-embed';
+
+export default function TechStackGraph() {
+  let url =
+    'http://underdog-devs-ds-a-dev.us-east-1.elasticbeanstalk.com/graph/tech-stack-by-role';
+
+  axios
+    .get(url)
+    .then(res => {
+      const result = embed('#vis', url);
```

Comment:
Let's figure out how to remove this linter warning from our repository. This is showing that Result has been defined but it is not being used.

Reply:
removed const result = to address the warning